Delete just grafana-cookies

PocketSquirrl opened the pull request:
Delete the grafana cookies and let normal cookies pass.
Added to #5457

manuquentin committed:
Avoid to request influxdb when query is hidden

bergquist committed:
docs(changelog): add note about cli fix #5410

bergquist committed:
style(light-theme): make text-color-weak a little bit darker

otherwise its not possible to see which queries are disabled
in the query editor

Soulou committed:
Fix node version in package.json

Value was incoherent since
https://github.com/grafana/grafana/commit/1900c81d9f9507ed4dbd18e95569480dfd0922f7

Commit:
Update debian.md

Commit:
Delete just grafana-cookies

Delete the grafana cookies and let normal cookies pass.

Commit:
Delete just grafana-cookies

Delete the grafana cookies and let normal cookies pass.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=5458) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=5458) before we can accept your contribution.<br/>

bergquist committed:
Merge pull request #5454 from Scalingo/node_version_package_json

Fix node version in package.json

mtanda committed:
add cloudwatch metrics and dimensions

mtanda committed:
reorder metrics/dimensions

Commit:
Check for empty cookie

Without the check this would fail if the cookie is empty

Commit:
Check for empty cookie

Without the check this would fail if the cookie is empty

tmc committed:
properly quote font-family on singlestat

bergquist committed:
Merge pull request #5465 from tmc/patch-1

properly quote font-family on singlestat

bergquist committed:
Merge pull request #5461 from mtanda/pr_cloudwatch_add_metrics

(cloudwatch) add metrics

bergquist committed:
Merge pull request #5455 from rakotomandimby/patch-1

Update debian.md to Debian Jessie

torkelo committed:
fix(modal): fixed duplicate confirm modal issue

torkelo committed:
fix(css): added responsive media query for phones that fix header wrap

stuartnelson3 committed:
[prometheus] Use `panelId` and `target.refId` for requestId

Using these two values ties requests to a
particular query position within a panel, ensuring
that requests are canceled if:

- Duplicate requests with the same query are sent
- Requests from the same query position (but a
different query) are sent

The last point is important as it identifies
queries by a physical location in the dashboard
instead of with the query expression.  If a
different query from the same position in a panel
is sent, the previous request should be canceled.

timmue commented:
I don't know why the checks are failing... I can't reproduce these at my own machine and thought the last commit would fix this.
Does anyone know whats wrong with my coding?

bergquist committed:
Merge pull request #5469 from stuartnelson3/create-unique-prometheus-req-id

[prometheus] Use `panelId` and `target.refId` for requestId

bergquist commented:
@timmue Please rebase onto master and reinstall your node_modules. That should fix it.

arthepsy committed:
Log to default log directory, only if custom configuration failed. (#5393)

bergquist committed:
Merge branch 'master' into influxdb_stop_hidden_queries

bergquist committed:
Merge branch 'master' into influxdb_stop_hidden_queries

bergquist committed:
Merge branch 'influxdb_stop_hidden_queries'

Commit:
Merge branch 'master' of https://github.com/timmue/grafana
